packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/engine/core.py

```
@@ -189,10 +189,10 @@ async def initialize(
                         vector_size = None
                 # 3) Deprecated fallback
                 if vector_size is None:
-                    vector_size = 1536
+                    vector_size = 1024
                     try:
                         self.logger.warning(
-                            "No vector_size provided via global.llm or env; falling back to 1536 (deprecated)."
+                            "No vector_size provided via global.llm or env; falling back to 1024 (deprecated)."
                         )
                     except Exception:
                         pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)